README.md

```diff
@@ -147,32 +147,52 @@ You can add your own template, just respect the naming rule : `template_name.tpl
 
 ## Testing the snippets locally
 
-It's possible to play test locally by running at the root of the project
+You can test the snippets locally by using the script `run-snippet-tests.sh`.  
+This script take the path of the snippets you want to test as argument and it will look recursively for snippets.  
+You must at least provide a sdk language and version in the provided path.  
```

Comment:
Can you provide a path example instead?

I guess it goes like that? 

`<path>/<language>/<version>/path/to/snippets`